README.md

```diff
@@ -1 +1,131 @@
-# wordvecspace
\ No newline at end of file
+# Word Vector Space Abstraction
+
+## Installation
+
+
+Install python-2.7
+
+sudo pip install numpy pandas numba
+sudo apt install libopenblas-dev
+
+### Usage of wordvecspace library
+
+$python
+
+#Import classes from wordvecspace library
```

Comment:
@RamanjaneyuluIdavalapati This line is marked as a level 1 header which is incorrect as per how markdown formatting is supposed to be used. Please fix.